probe/endpoint/ebpf.go

```diff
@@ -84,6 +84,31 @@ func isKernelSupported() error {
 		return fmt.Errorf("got kernel %s but need kernel >=4.4", release)
 	}
 
+	if strings.Contains(version, "Ubuntu") {
+		// Check for specific Ubuntu kernel versions with
+		// known issues.
+
+		// The vendor specific release part could look like:
+		// '<ABI number>.<upload number>-<flavour>' or
+		// '<ABI number>-<flavour>'
+		// See https://wiki.ubuntu.com/Kernel/FAQ
+
+		// cut away flavour
+		vendorStr := strings.Split(releaseParts[4], "-")[0]
+		// cut away upload number
+		abiNumberStr := strings.Split(vendorStr, ".")[0]
+		abiNumber, err := strconv.Atoi(abiNumberStr)
+		if err != nil {
+			return err
+		}
+		// TODO: give the check an upper limit once the bug is fixed
+		if abiNumber >= 119 {
+			// https://github.com/weaveworks/scope/issues/3131
+			// https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454
```

Comment:
I'm guessing the fix in the Ubuntu kernel is to be released soon. Should we wait until it is released, so we know the exact abiNumber range where it's broken?

Reply:
> I'm guessing the fix in the Ubuntu kernel is to be released soon.

Any idea how long that might take?

Frankly, if there is going to be a new LTS security-updates kernel release with the fix in the next few days, I am not sure it's even worth bothering having any check here at all.

Reply:
>> I'm guessing the fix in the Ubuntu kernel is to be released soon.

> Any idea how long that might take?

We can't tell and have asked Seth (Ubuntu) if he can give an ETA.

> Frankly, if there is going to be a new LTS security-updates kernel release with the fix in the next few days, I am not sure it's even worth bothering having any check here at all.

I agree.

Reply:
Update: approximately around 4 weeks for the update to be published, https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454/comments/6